ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -26,6 +26,15 @@ Defines the app components and versions to configure in the application workspac
   For Health (standalone) the default is:
     `health=latest`
 
+### mas_app_optimizer_plan
```

Comment:
This should just be `mas_app_ws_plan` .. optimizer is the first app to support more than one install plan.  Let's word the documentation that this is a generic thing, the same way `mas_app_ws_spec` works, but is only used by manage at the moment.

Reply:
I will update to be `mas_app_plan` as this is a Application level parameter 👍